### PR TITLE
media-types: Fix broken links

### PR DESCRIPTION
**Pull request by coolljt0725**
Related to https://github.com/distribution/distribution/pull/4101, This PR move `spec/manifest-v2-2.md` to `content/spec/manifest-v2-2.md`

### CODE REVIEW

File: media-types.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Similar/related schema:
 
 Similar/related schema:
 
-- [application/vnd.docker.distribution.manifest.v2+json](https://github.com/distribution/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-field-descriptions)
+- [application/vnd.docker.distribution.manifest.v2+json](https://github.com/distribution/distribution/blob/master/docs/content/spec/manifest-v2-2.md#image-manifest-field-descriptions)
```
Comment (sudo-bmitch) [inline media-types.md:54]:
```suggestion
- [application/vnd.docker.distribution.manifest.v2+json](https://github.com/distribution/distribution/blob/v2.8.3/docs/spec/manifest-v2-2.md#image-manifest)
```

**Review by sudo-bmitch — CHANGES_REQUESTED** [review]
My preference is to depend on a tag rather than the master branch. Once the 3.0 release goes GA there, we can adjust the tags and paths together. It also looks like the image manifest anchor was been wrong so this is a good opportunity to fix that too.

File: media-types.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ This section shows where the OCI Image Specification is compatible with formats
 
 Similar/related schema:
 
-- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/distribution/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list)
+- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/distribution/distribution/blob/master/docs/content/spec/manifest-v2-2.md#manifest-list)
```
[the remark this answers is not in the document]
Comment (coolljt0725) [reply media-types.md:45]: Good suggestion, agree. Thanks

**Comment by coolljt0725** [comment]
updated according to @sudo-bmitch 's suggestions.

**Review by sudo-bmitch — APPROVED** [review]
LGTM. I'd recommend holding off on the final merge until after the 1.1.0 vote completes.

**Review by tianon — APPROVED** [review]

**Pull request merged**